Doc/Bib/README.md

```diff
@@ -0,0 +1,109 @@
+# The GAP bibliography
+
+The GAP bibliography resides in the directory `GapWWW/Doc/Bib`.
+
+
+## Data files
+
+- `GapCite.MR`: this file contains just MR numbers of papers that cite
+or refer to (one of the versions of) GAP (here and below "MR" stands
+for "Mathematical Reviews".). The format is alternatingly one line of
+the form 1stAuthorSurname Paper (not starting with a blank) and one
+line MR-Number (starting with a blank). MR numbers will be used to get
+full bibliographic info from MathSciNet, and the textual description
+only helps when adding papers to the file (in particular, to keep
+entries sorted by the first author).
```

Comment:
This goes beyond this PR, but I really wonder about the logic behind this format: Wouldn't it be much simpler to just store the MR numbers, sorted numerically? That makes it trivial to detect and avoid duplicates?

Reply:
Long story short - this format was used long before I took over the Doc/Bib, and I never came to changing it, and last time I updated the bibliography, I told myself to not to start that, because that all should be rewritten in Python...

Reply:
Initially, someone was maintaining it manually, and I think that helped them to navigate somehow. Check for updates script produces already properly formatted lines for this file for copying and pasting.

Reply:
Sure, I just wanted to mention it. If you have time for a full rewrite, fine. Otherwise, though I'd be careful not to fall into the trap to never touch the code again because it might *perhaps* be rewritten one day... But that's just a remark, I'll not try to dictate how you maintain this code :-)